src/core/compiler.jl

```diff
@@ -121,10 +120,10 @@ function _tilde(vsym, left, dist, model_info)
     main_body_names = model_info[:main_body_names]
     model_name = main_body_names[:model]
 
+    model_info[:tent_dvars_list] = copy(model_info[:arg_syms])
```

Comment:
What does `tent_dvars_list` short for?

Reply:
Tentative data variables list. It is basically the arguments unless they are passed as `missing` later when constructing the models. The `findfirst` lines are useless, need to be removed.